src/sentry/services/hybrid_cloud/auth/__init__.py

```diff
@@ -71,6 +151,10 @@ class ApiMemberSsoState:
 
 @dataclass
 class AuthenticationRequest:
+    # HTTP_X_SENTRY_RELAY_ID
```

Comment:
@mikejihbe Actually, can you remove this logic?  I realized after the fact that we never need relay logic in the hybrid cloud layer, but I forgot to remove its references from this file.

Reply:
Or maybe not, actually unsure.  Nevermind!  Leave it, I may wire it up later if it becomes necessary.